packages/patternfly-4/react-core/src/layouts/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/layouts/Gallery/gallery';
+
+export interface GalleryProps extends React.HTMLProps<HTMLDivElement> {
+  /** content rendered inside the Gallery layout */
+  children?: React.ReactNode; 
+  /** additional classes added to the Gallery layout */
+  className?: string; 
+  /** Adds space between children. */
+  gutter?: boolean; 
```

Comment:
This does not make sense form previous implementation of gallery.  Shouldn't it be a union? `sm|md|lg`?

why is gutter now a boolean.  This is a breaking change.

Reply:
Currently sm, md, lg doesn't do anything other than resolve to true. Based on conversation, I thought we weren't going to continue passing the string and make it a boolean, but I can change them all back to sm, md, lg and just change them again later if and when we choose to fix the gutter sizes.